core/test/integration/CMakeLists.txt

```diff
@@ -33,4 +33,112 @@ target_include_directories(ledger-core-integration-tests PUBLIC ../../../qt-host
 include(CopyAndInstallImportedTargets)
 copy_install_imported_targets(ledger-core-integration-tests crypto ledger-core)
 
-add_test (NAME ledger-core-integration-tests COMMAND ledger-core-integration-tests)
\ No newline at end of file
+#add_test (NAME ledger-core-integration-tests COMMAND ledger-core-integration-tests)
```

Comment:
Is there a way of launching a batch without breaking CI (Like refacto some with a *) ?

Reply:
I don’t get what you’re asking for. Do you mean a way to by-pass the tests?